WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -17,7 +17,7 @@
         <% else %>
           <%= icon('hourglass-start', title: "Starts in #{ pluralize((competition.start_date - Date.today).to_i, "day") }", data: { toggle: "tooltip" }) %>
         <% end %>
-        <%= wca_date_range(competition.start_date, competition.end_date)  %>
+        <%= competition.start_date && competition.end_date ? wca_date_range(competition.start_date, competition.end_date) : "Unscheduled" %>
```

Comment:
That is true, unscheduled competitions shouldn't end up in @competitions but if they did it would throw an error. I thought it was better to put this in as a safeguard. Happy to remove it if you think it's not required.